src/main.rs

```diff
@@ -62,8 +62,8 @@ async fn main() -> Result<()> {
             (@subcommand push =>
              (about: "Pushes a Kubewarden policy to an OCI registry")
              (@arg ("docker-config-json-path"): --("docker-config-json-path") +takes_value "Path to a Docker config.json-like path. Can be used to indicate registry authentication details")
-             (@arg ("policy-path"): * -p --("policy-path") +takes_value "Policy file to push")
              (@arg ("sources-path"): --("sources-path") +takes_value "YAML file holding source information (https, registry insecure hosts, custom CA's...)")
+             (@arg ("policy"): * "Policy to push. Can be the path to a local file, or a policy URI")
              (@arg ("uri"): * "Policy URI. Supported schemes: registry://")
              (@arg ("force"): -f --("force") "push also a policy that is not annotated")
```

Comment:
I wonder if we should keep named arguments alphabetically sorted. `--force` was already off though.

Reply:
I can sort the flag alphabetically